src/views/RegionAdditionalInfo/styles.module.css

```diff
@@ -0,0 +1,32 @@
+.region-preparedness {
```

Comment:
This should match the name of component
```suggestion
.region-additional-info {
```